.github/workflows/testing.yml

```diff
@@ -108,6 +108,7 @@ jobs:
           go test ${{ matrix.testflags }} -cpu 1,4 -timeout 7m google.golang.org/grpc/...
           cd ${GITHUB_WORKSPACE}/security/advancedtls && go test ${{ matrix.testflags }} -timeout 2m google.golang.org/grpc/security/advancedtls/...
           cd ${GITHUB_WORKSPACE}/security/authorization && go test ${{ matrix.testflags }} -timeout 2m google.golang.org/grpc/security/authorization/...
+          cd ${GITHUB_WORKSPACE}/gcp/observability && go test ${{ matrix.testflags }} -timeout 2m google.golang.org/grpc/gcp/observability/...
```

Comment:
Would this work to make sure this oversight doesn't happen again?

```sh
cd "${GITHUB_WORKSPACE}" # ? I don't think we need this
for MOD_FILE in $(find . -name 'go.mod' | grep -v './go.mod'); do
  pushd "$(dirname ${MOD_FILE})"
  go test ${{ matrix.testflags }} -timeout 2m ./...
  popd
done
```

Reply:
Updated.

I kept the first test line:

```
go test ${{ matrix.testflags }} -cpu 1,4 -timeout 7m google.golang.org/grpc/...
```

It adds a CPU mask, I guess it's somewhat special (and the suggestion script explicitly excluded it).